public/css/game.css

```diff
@@ -0,0 +1,141 @@
+.container {
+  width: 100%;
+  margin: 0 !important; }
```

Comment:
why are you using `!important`? You may want to try the concept of `specificity in css` over `!important rule`. In this case, having an `id` attached to the element in question and then adding styles to id selector makes the rule apply rather than using `!important` in that class.